### PR TITLE
Define Bazel external repository for DPDK SDE

### PR DESCRIPTION
**Pull request by ffoulkes**
Signed-off-by: Derek G Foster <derek.foster@intel.com>

### CODE REVIEW

File: bazel/external/dpdk.BUILD [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "dpdk_sde",
+    srcs = glob([
+        "dpdk-bin/lib/libbf_switchd_lib.so*",
+        "dpdk-bin/lib/libdriver.so",
+        "dpdk-bin/lib/libtarget_sys.so",
```
Comment (nupuruttarwar) [inline bazel/external/dpdk.BUILD:20]: Do we need target_utils as well here?

Comment (nupuruttarwar) [reply bazel/external/dpdk.BUILD:20]: Similar question about dpdk library, earlier we linked with dpdk_infra

Comment (ffoulkes) [reply bazel/external/dpdk.BUILD:20]: These definitions are sufficient for `bazel build //stratum/hal/bin/tdi/dpdk:stratum_dpdk --define target=dpdk` to complete without error.

Comment (ffoulkes) [reply bazel/external/dpdk.BUILD:20]: I just verified that I can build `tdi_pipeline_builder` and `gnmi_cli` as well.

We might need to make some additions or changes later, but what we have appears to be sufficient for our immediate needs.

**Review by nupuruttarwar — APPROVED** [review]
LGTM

**Pull request merged**